src/Symfony/Component/Console/Tests/DependencyInjection/AddConsoleCommandPassTest.php

```diff
@@ -9,9 +9,9 @@
  * file that was distributed with this source code.
```

Comment:
The deprecated pass must still be tested (the test will be removed at the same time than the class).